components/lib/cascadeselect/CascadeSelect.js

```diff
@@ -235,28 +235,28 @@ export const CascadeSelect = React.memo(
                 const selector = `${attributeSelectorState}_panel`;
                 const innerHTML = `
 @media screen and (max-width: ${props.breakpoint}) {
-    .p-cascadeselect-panel[${selector}] .p-cascadeselect-items-wrapper > ul {
```

Comment:
Uusually these `p-` classes are left like this so they can be overridden by the PrimeReact SASS Themes: https://github.com/primefaces/primereact-sass-theme/blob/main/theme-base/components/input/_cascadeselect.scss

Reply:
Do you think it can be modified like this?
```js
isUnstyled() ? [data-pc-section="panel"][${selector}] : .p-cascadeselect-panel[${selector}]
```

Reply:
If the breakpoints need to use the className,
I should revert className in  #5586 and #5532

Reply:
@melloware  
I have reverted
Do you think unstyled mode should be implemented breakpoints by the user?


> I'm not sure if in unstyled mode, Is this logic should be implemented by the user?
> 
> Do you think it should be modified like this?
> 
> ```tsx
> isUnstyled()  ? [data-pc-section="panel"][${selector}] : .p-cascadeselect-panel[${selector}]
> ```

Reply:
No in the Dialog fix its OK because its creating an inline width style.

Reply:
Thank you for your review!
I will submit a PR to revert the breakpoints of Datatable